pom.xml

```
@@ -3,11 +3,11 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.40</version>
+    <version>4.51</version>
```

```suggestion
    <version>4.53</version>
```
This or changing the PR title 🙂